src/engine/strat_engine/pool.rs

```diff
@@ -191,9 +192,10 @@ impl StratPool {
     pub fn setup(
         uuid: PoolUuid,
         devnodes: &HashMap<Device, PathBuf>,
+        timestamp: DateTime<Utc>,
         metadata: &PoolSave,
     ) -> StratisResult<(Name, StratPool)> {
-        let mut backstore = Backstore::setup(uuid, &metadata.backstore, devnodes, None)?;
+        let mut backstore = Backstore::setup(uuid, &metadata.backstore, devnodes, Some(timestamp))?;
```

Comment:
I think that it is possible to make ```Backstore::setup```'s timestamp parameter not an ```Option```. The argument is that ```setup``` takes a ```BackstoreSave``` argument, which is a component of the variable length metadata. Since that is not an ```Option```, then the variable length metadata must have been written, hence have a timestamp, so the timestamp is not an ```Option``` here.

Reply:
Let me take a look. I just was hesitant in case it was initialized with `None` elsewhere in the case of initialization of metadata on a device. If that's the only place it's used, I'll remove the `Option` here.

Reply:
Interesting. That does seem to be the only place we have to worry about that call. I'll remove the `Option`.